src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -64,7 +63,6 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
             } : { 'data-ouia-safe': false } }
         >
             <TemplateCardHead>
```

Comment:
@ryelo i made changes and tried to push them up but its not working. Just remove the `<TemplateCardHead>` wrapper, and then also the styles you added

Reply:
Awesome! Will do!